Makefile

```diff
@@ -15,9 +22,3 @@ build_docs:
 
 changelog:
 	@echo "Not supported any more. Run ./generate_changelog.py <version_number> instead!"
-
-sass_watch:
```

Comment:
Pretty sure these are now unused, figured it would be OK to yank them out

Reply:
Yes you're right, sorry I missed those when removing the legacy UI :+1: